app/controllers/users_controller.rb

```diff
@@ -67,9 +67,8 @@ def create_external
     @user.password = generate_new_password
 
     if @user.save
-      # send email
-      Notifier.delay.new_user(user: @user, password: @user.password)
-      redirect_to facility_users_path(user: @user.id)
+      @user.create_default_price_group!
+      save_user_success
```

Comment:
The existing `save_user_success` generates the mailer with`password: nil`. It might be both internal and external can use `password: @user.password` (I suspect netid users have a `nil` password field).